[2021-03-09 09:44:18,030] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: extract_standings.load_standings 2021-02-08T00:00:00+00:00 [queued]>
[2021-03-09 09:44:18,035] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: extract_standings.load_standings 2021-02-08T00:00:00+00:00 [queued]>
[2021-03-09 09:44:18,035] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 09:44:18,035] {taskinstance.py:1043} INFO - Starting attempt 1 of 3
[2021-03-09 09:44:18,035] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-09 09:44:18,040] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): load_standings> on 2021-02-08T00:00:00+00:00
[2021-03-09 09:44:18,043] {standard_task_runner.py:52} INFO - Started process 25153 to run task
[2021-03-09 09:44:18,048] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'extract_standings', 'load_standings', '2021-02-08T00:00:00+00:00', '--job-id', '82', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/standings_dag.py', '--cfg-path', '/var/folders/n1/tvvct6mn75xbwwwl4s02tfdc0000gn/T/tmpcym_x141', '--error-file', '/var/folders/n1/tvvct6mn75xbwwwl4s02tfdc0000gn/T/tmpy589cxbb']
[2021-03-09 09:44:18,050] {standard_task_runner.py:77} INFO - Job 82: Subtask load_standings
[2021-03-09 09:44:18,079] {logging_mixin.py:104} INFO - Running <TaskInstance: extract_standings.load_standings 2021-02-08T00:00:00+00:00 [running]> on host 25.1.168.192.in-addr.arpa
[2021-03-09 09:44:18,103] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=AWTY
AIRFLOW_CTX_DAG_ID=extract_standings
AIRFLOW_CTX_TASK_ID=load_standings
AIRFLOW_CTX_EXECUTION_DATE=2021-02-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-08T00:00:00+00:00
[2021-03-09 09:44:18,104] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/n1/tvvct6mn75xbwwwl4s02tfdc0000gn/T
[2021-03-09 09:44:18,104] {bash.py:158} INFO - Running command: curl --header "Content-Type: application/json" --request POST --data @./standings_dump.json http://127.0.0.1:4321/db/update
[2021-03-09 09:44:18,111] {bash.py:169} INFO - Output:
[2021-03-09 09:44:18,116] {bash.py:173} INFO - Warning: Couldn't read data from file "./standings_dump.json", this makes an
[2021-03-09 09:44:18,116] {bash.py:173} INFO - Warning: empty POST.
[2021-03-09 09:44:18,117] {bash.py:173} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2021-03-09 09:44:18,117] {bash.py:173} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2021-03-09 09:44:18,119] {bash.py:173} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0100   192  100   192    0     0  96000      0 --:--:-- --:--:-- --:--:-- 96000
[2021-03-09 09:44:18,119] {bash.py:173} INFO - <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
[2021-03-09 09:44:18,120] {bash.py:173} INFO - <title>400 Bad Request</title>
[2021-03-09 09:44:18,120] {bash.py:173} INFO - <h1>Bad Request</h1>
[2021-03-09 09:44:18,120] {bash.py:173} INFO - <p>The browser (or proxy) sent a request that this server could not understand.</p>
[2021-03-09 09:44:18,121] {bash.py:177} INFO - Command exited with return code 0
[2021-03-09 09:44:18,132] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=extract_standings, task_id=load_standings, execution_date=20210208T000000, start_date=20210309T154418, end_date=20210309T154418
[2021-03-09 09:44:18,141] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-09 09:44:18,147] {local_task_job.py:146} INFO - Task exited with return code 0
